service/frontend/workflowHandler.go

```diff
@@ -2238,7 +2238,9 @@ func (wh *WorkflowHandler) error(err error, scope int) error {
 	case *gen.InternalServiceError:
 		logging.LogInternalServiceError(wh.Service.GetLogger(), err)
 		wh.metricsClient.IncCounter(scope, metrics.CadenceFailures)
-		return err
+		// NOTE: For internal error, we won't return thrift error from cadence-frontend.
```

Comment:
what user see on client side for this error?

Reply:
as discussed, client is not using it for determining retry: https://github.com/uber-go/cadence-client/blob/master/internal/internal_task_pollers.go#L150

Reply:
so we just let the client deal with unknown error?